warema_wms/shade.py

```diff
@@ -9,7 +9,7 @@
 
 class Shade:
     def __init__(self, wms_ctrl: WmsController, room, channel,
-                 time_between_cmds=0.1, num_retries=3, position=0, is_moving=False):
+                 time_between_cmds=0.1, num_retries=3, position=0, tilt_position=0, is_moving=False):
```

Comment:
```suggestion
                 time_between_cmds=0.1, num_retries=3, position=0, tilt_position=30, is_moving=False):
```

If vaild values range between 30-100 the default should also be between those values.

Reply:
ah, i understand, this was caused by my lack of understanding, sorry

Reply:
so this means if I set 100 here as default, the blinds move with this value until it is changed?

just for my unverstandig, would fit my personal needs,  could be documented so anybody else can set it regarding to his needs?
Warema default is tilt fully open, which imho is the most stupid solution for a heat and sun protection...

Reply:
This is only the initial value when the library (and therefore the class) is being initialized. Normally the tilt value would be passed from HA via the integration.